--- libshaderc_util/src/resources.cc	2023-05-21 15:20:53.530259500 +0100
+++ libshaderc_util/src/resources.cc	2023-05-12 21:25:18.908605600 +0100
@@ -125,6 +125,15 @@
     /* .maxTaskWorkGroupSizeY_NV = */ 1,
     /* .maxTaskWorkGroupSizeZ_NV = */ 1,
     /* .maxMeshViewCountNV = */ 4,
+    /* maxMeshOutputVerticesEXT = */ 1,
+    /* maxMeshOutputPrimitivesEXT = */ 1,
+    /* maxMeshWorkGroupSizeX_EXT = */ 1,
+    /* maxMeshWorkGroupSizeY_EXT = */ 1,
+    /* maxMeshWorkGroupSizeZ_EXT = */ 1,
+    /* maxTaskWorkGroupSizeX_EXT = */ 1,
+    /* maxTaskWorkGroupSizeY_EXT = */ 1,
+    /* maxTaskWorkGroupSizeZ_EXT = */ 1,
+    /* maxMeshViewCountEXT = */ 1,
     /* .maxDualSourceDrawBuffersEXT = */ 1,
     // This is the glslang TLimits structure.
     // It defines whether or not the following features are enabled.
